package.json

```diff
@@ -52,5 +52,8 @@
 		"webpack-cli": "^3.3.6",
 		"webpack-merge": "^4.2.2"
 	},
-	"license": "MIT"
+	"license": "MIT",
+	"files": [
+		"dist/floating-focus.js"
```

Comment:
I think here you want to just make this `/dist`, so that any other files put in there are also automatically included.
Note the leading `/`: from what I'm reading on the interwebz this is important to not accidentally include nested folders (not really relevant for this but oh well)